src/Angor/Client/Pages/Create.razor

```diff
@@ -1172,4 +1175,45 @@
         isPresetMode = !isPresetMode; // Toggle between Presets and Automatic
         StateHasChanged(); // Update the UI
     }
+    
+    
+    private void SetDefaultImagesForTestnet(ProjectMetadata metadata)
+    {
+        var random = new Random();
+        if (network.NetworkType == NetworkType.Testnet)
+        {
+            if (string.IsNullOrEmpty(metadata.Banner))
+            {
+                metadata.Banner = spacePhotos[random.Next(spacePhotos.Length)];
+            }
+
+            if (string.IsNullOrEmpty(metadata.Picture))
+            {
+                metadata.Picture = spacePhotos[random.Next(spacePhotos.Length)];
+            }
+        }
+    }
+    
+    private readonly string[] spacePhotos = new[]
+    {
+        "https://images-assets.nasa.gov/image/hubble-observes-one-of-a-kind-star-nicknamed-nasty_17754652960_o/hubble-observes-one-of-a-kind-star-nicknamed-nasty_17754652960_o~thumb.jpg?as=webp",
```

Comment:
Are we allowed to use these images freely?

Reply:
as far as i read from here, it seems like we can.

https://www.nasa.gov/nasa-brand-center/images-and-media/